radboudumc-template/templates/new/publications-author.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
```

Comment:
Could we also show publications for authors per year? Maybe we can add a flag to their members page "publications_per_year: true".

Reply:
sure, we can make a new issue for this if the new bib parser is in use